mrtt-ui/src/components/SiteInterventions.js

```diff
@@ -160,3 +302,9 @@ function SiteInterventionsForm() {
 }
 
 export default SiteInterventionsForm
+
+const InnerFormDiv = styled('div')`
```

Comment:
Is this used? Wouldnt it have to be above the code that uses it since ist `const` and doesnt get hoisted like `var` does?